### ISSUE TITLE
Proposal for lists

### ISSUE THREAD

**Issue by justjosias** [nileshtrivedi/better#26]
Choosing what to recommend can be a tricky subject, and the pros and cons of various options are more than can be expressed in a small popup. Thus I believe that it would be better to use a third-party list of replacements. That way you can link to a more complete explanation and simply worry about the browser extension rather than the politics of recommending certain replacements. It also allows the users to more reasonably consider the alternatives instead of making a judgement based on a short phrase. 

I like how this extension often recommends switching.software. I believe this would be a good precedent to extend to the other recommendations as well.

**Comment by nileshtrivedi** [comment]
Yes, this is an important point. The pop-up can just have on short sentence to pique the user's interest in the alternative - which may not be enough. I find myself leaning more & more towards link to an explanatory page rather than a single alternative product. Fortunately, this is completely driven by the list and we can iterate to find the best approach.

@justjosias Thanks for stopping by! Appreciateit.
